Applications/ConsoleReferenceClient/UAClient.cs

```diff
@@ -505,10 +491,12 @@ private void CertificateValidation(CertificateValidator sender, CertificateValid
 
         private ApplicationConfiguration m_configuration;
 
-        private string m_serverUrl = "opc.tcp://localhost:62541/Quickstarts/ReferenceServer";
-
         private Session m_session;
 
+        private readonly IOutput _output;
+
+        private readonly Action<IList, IList> _validateResponse;
+
```

Comment:
to respect the naming convention if should be `m_output `and `_validateResponse`

Reply:
I changed those private members to have `m_` prepended to the original names.

Reply:
@lkomanetz ,
I'll merge your updates now.
Thank you!